packages/dev_disc/README.md

```diff
@@ -64,9 +64,9 @@ devsim_cmd_msg_pipe = /var/tmp/liota/devsim_cmd_messenger.fifo	# the named pipe
 
 							# if no item in this list, Device Discovery will not be started
 
-disc_msg_pipe = /var/tmp/liota/discovery_messenger.fifo		# currently, only support these 4 types
+disc_msg_pipe = /var/tmp/liota/discovery_messenger.fifo		# currently, support these 4 types (currently,
```

Comment:
Is the address correct? It seems that it was changed in other locations but not here.